modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -0,0 +1,183 @@
+// Module included in the following assemblies:
+// * observability/network_observability/netobserv-cli-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-netobserv-cli-commands_{context}"]
+= Network Observability CLI reference 
+The Network Observability CLI (`oc netobserv`) is a CLI tool for capturing flow data and packet data for further analysis.
+
+.`oc netobserv` syntax
+[source,terminal]
+----
+$ oc netobserv [<command>] [<feature_command>] [<subcommands>] <1>
+----
+1. Feature commands can only be used with the `oc netobserv flows`. They cannot be used with `oc netobserv packets`.
+
+[cols="3a,8a",options="header"]
+.Basic commands
+|===
+|Option| Description
+
+| `flows`
+| Capture flows information. For subcommands, see the "Flows subcommands" table on this page. 
+
+| `packets`
+| Capture packets from a specific protocol or port pair, such as 'netobserv packets --filter=tcp,80'. For more information about packet capture, see the "Packet capture subcommand" section on this page.
+
+| `cleanup`
+| Remove netobserv components.
+
+| `version`
+| Print the software version.
+
+| `help`
+| Show help.
+|===
+
+== Feature commands
+The Network Observability Operator features can only be enabled when capturing flows. They do not work with packet capture.
+
+.Syntax
+[source,terminal]
+----
+$ oc netobserv flows [<feature_command>] [<subcommands>]
+----
+
+.Network Observability feature enablement commands
+|===
+|Option| Description| Possible values| Default
+
+| `--help`
+| Show help
+| N/A
+| N/A
+
+| `--enable_pktdrop` 	
+| Enable packet drop
+| `true`, `false`
+| `false`
+
+| `--enable_rtt`
+| Enable round trip time 	
+| `true`, `false` 	
+| `false`
+
+| `--enable_dns`
+| Enable DNS tracking 	
+| `true`, `false` 	
+| `false`
+
+| `--interfaces`
+| Interfaces to match on the flow e.g., "eth0,eth1" 	
+| "<interface>"
+|===
+
+== Flow capture subcommands
+Flow capture has mandatory commands and optional commands. 
+
+.`oc netobserv flows` syntax
+[source,terminal]
+----
+$ oc netobserv flows [<feature_command>] [<subcommands>]
+----
+
+.Flow capture subcommands
+|===
+|Option| Description| Possible values| Mandatory| Default
```

Comment:
A few of the same commands from the above tables could be applied here as well

Reply:
You can use the feature options with flow capture, if that's what you mean. so yeah I should note it here, since I noted it above. Good thing to notice.